### DISCUSSION TITLE
Theme from javascript

### DISCUSSION THREAD

**Discussion by smurfb** [saadeghi/daisyui#2640] [General]
Hi!

Is there anyway to know what theme that is selected in the theme-controller, no the colours but perhaps the name? I want to change  background image according to what theme is used. Or any other way. Using svelte. 

Thanks for daisyui, its awsome!

**Comment by saadeghi** [comment]
For checkbox: https://svelte.dev/repl/ac2ac981c07a4e75ad77a489dc49f950?version=4.2.8  
For radio inputs: https://svelte.dev/repl/3539f033996149499d66c48e03e1a518?version=4.2.8  


Let me know if you have a question.
